admin/src/components/DeletedUserFormular.vue

```diff
@@ -34,9 +51,34 @@ export default {
   data() {
     return {
       checked: false,
+      modalTitle: '',
+      modalQuestion: '',
+      modalOkTitle: '',
+      modalEvent: null,
+      username: '',
+      variant: 'danger',
     }
   },
   methods: {
+    showModal(type) {
+      this.username = `${this.item.firstName} ${this.item.lastName}`
+
+      if (type === 'deleteUser') {
+        this.variant = 'danger'
+        this.modalTitle = this.$t('overlay.deleteUser.title')
+        this.modalQuestion = this.$t('overlay.deleteUser.question', { username: this.username })
+        this.modalOkTitle = this.$t('overlay.deleteUser.yes')
+        this.modalEvent = this.deleteUser
+      }
+
+      if (type === 'undeleteUser') {
+        this.variant = 'success'
+        this.modalTitle = this.$t('overlay.undeleteUser.title')
+        this.modalQuestion = this.$t('overlay.undeleteUser.question', { username: this.username })
+        this.modalOkTitle = this.$t('overlay.undeleteUser.yes')
+        this.modalEvent = this.unDeleteUser
+      }
+    },
```

Comment:
I have no further insight into this - but for me this looks like you just want to define two modals, each with the correct values submitted and open them accordingly.

The whole mechanic behind `v-b-modal` seems to support my statement here I think - it seems like the whole mechanic is made to allow to define multiple modals which then open when you call `showModal` with the specific reference defined.
Reference: https://bootstrap-vue.org/docs/components/modal#using-v-b-modal-directive

And here is a multi-modal example in the docu: https://bootstrap-vue.org/docs/components/modal#multiple-modal-support

Reply:
This is resolved by utilizing the `bvModal.msgBoxConfirm()` method.